critiquebrainz/db/revision.py

```diff
@@ -148,23 +152,34 @@ def get_revision_number(review_id, revision_id):
     return rev_num
 
 
-def create(review_id, text):
+def create(review_id, text, rating):
```

Comment:
Even if one of text or rating is None, both of them must be passed to insert their values. And, the check if text and rating both are None is placed at the start of update and create in review. So, it works fine.